solutions/13-14/script.js

```diff
@@ -0,0 +1,43 @@
+const ul = document.querySelector(".authors");
+const url = "https://randomuser.me/api/?results=4";
+
+function createNode(element) {
+  return document.createElement(element);
+}
+
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+function createBusinessCard(person) {
+  const { name, location, email, phone } = person;
```

Comment:
```
const { name, location, email, phone } = person;
const { title, first, last } = name;
const { street, city, postcode} = location;
```
You can make more advance destructurization 
```
const {
name: { title, first, last }, 
location: { street, city, postcode},
email,
phone,
} =person;
```
And also you can do it already in an arguments section
```
function createBusinessCard( {
name: { title, first, last }, 
location: { street, city, postcode},
email,
phone,
})
```